index.bs

```diff
@@ -525,7 +525,7 @@ urlPrefix: https://www.w3.org/2017/Process-20170301/; spec: W3C-PROCESS
 <section>
   <h2 id="algorithms">Algorithms</h2>
 
-  <h3 id="is-origin-trustworthy">
+  <h3 dfn-type="algorithm" export id="is-origin-trustworthy">
```

Comment:
Bikeshed doesn't like the `dfn-type="algorithm"` annotation.

Reply:
sorry, I always get confused - the type is "abstract-op" not algorithm; not sure why my local bikeshed didn't complain though